### ISSUE TITLE
Water masking

### ISSUE THREAD

**Issue by Artpom** [stackTom/FSEarthTiles#24]
Some of my areas are affected with wrong masking, so i have trees in the water, as can see in the picture - Gotland area ( Eastern part )
Is there any way to get rid trees out of the water?

![Screenshot 2024-03-06 191256](https://github.com/stackTom/FSEarthTiles/assets/161880250/d50b3209-a263-4186-bfcc-1fc5a6211c66)
![Screenshot 2024-03-06 191649](https://github.com/stackTom/FSEarthTiles/assets/161880250/2e5f8a5a-576f-4494-8b5b-f74bf2fca27f)

**Comment by stackTom** [comment]
The trees are due to the scenproc script and the generated .agn files. Feel free to remove the .agn files and it will fix the trees. The scenproc scripts can be improved to fix issues like this. Unfortunately I don't know much about scenproc, so I just included some scripts a community member made. Feel free to ask on the internet or on fsdeveloper forums for other scripts which won't have this issue.

**Comment by Artpom** [comment]
after removal of .agn files i have lost all P3D generated trees - i think that was not a solution i was looking for ;-)

**Comment by stackTom** [comment]
Yes, the agn files contain the trees. You need to modify the default scenproc scripts to not do this. I can ask someone knowledgeable in scenproc scripts to see if they can help, but I don't know how to use scenproc. Best place to ask would be fsdeveloper, where the creator of scenproc is active (Arno).

**Comment by Artpom** [comment]
Been there and see that water masking is a kind of hot topic but without any good solution so far.

**Comment by stackTom** [comment]
Let me ask the creators of the sceneproc script included in fset and see if they know of a solution.
